crowdstrike/table_crowdstrike_intel_actor.go

```diff
@@ -0,0 +1,193 @@
+package crowdstrike
+
+import (
+	"context"
+
+	"github.com/crowdstrike/gofalcon/falcon"
+	"github.com/crowdstrike/gofalcon/falcon/client"
+	"github.com/crowdstrike/gofalcon/falcon/client/intel"
+	"github.com/crowdstrike/gofalcon/falcon/models"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+func tableCrowdStrikeIntelActor(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "crowdstrike_intel_actor",
+		Description: "Threat intelligence actors.",
```

Comment:
Is there any more info we can add here about what threat intelligence actors are from CrowdStrike docs, resource descriptions, or articles?

Reply:
Please see comment above